HOW_WE_USE_GITHUB.md

```diff
@@ -272,7 +272,7 @@ please post details to the [Nucleus forums](https://community.anaconda.cloud/).
 <pre>
 Hi [@username],
 
-Thanks for voicing your concern about the performance of our dependency solver. To fix this, our official recommendation is using the "conda-libmamba-solver" instead of the default "classic" solver (more information about the "conda-libmamba-solver" can be found here: https://conda.github.io/conda-libmamba-solver/getting-started/).
+Thanks for voicing your concern about the performance of our legacy dependency solver. To fix this, our official recommendation is using the new default "conda-libmamba-solver" instead of the legacy "classic" solver (more information about the "conda-libmamba-solver" can be found here: https://conda.github.io/conda-libmamba-solver/getting-started/).
```

Comment:
```suggestion
Thanks for voicing your concern about the performance of the classic dependency solver. To fix this, our official recommendation is using the new default "conda-libmamba-solver" instead of the classic solver (more information about the "conda-libmamba-solver" can be found here: https://conda.github.io/conda-libmamba-solver/getting-started/).
```

FWIW we can't call the "classic" solver legacy yet, since there will be many people relying on it in the future and we shouldn't rush to imply that it'll be deprecated